### ISSUE TITLE
[CoE Starter Kit - BUG] BYODL-BYODL PowerBI dashboard missing data

### ISSUE THREAD

**Issue by jayakumarkgds** [microsoft/coe-starter-kit#5812]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Suddenly from last week all the information in powerBi dasboprd is missing.Then i tried new data flow and new template still data is not populating in powerBI.Ther is no error in refresh for powerBi dashboard
I can see the data in dataflow refresh status.but same is not visible n dashboard.
![image](https://github.com/microsoft/coe-starter-kit/assets/87489417/3cd29d38-d578-480b-ba19-6192ff3be6a3)

![image](https://github.com/microsoft/coe-starter-kit/assets/87489417/ffa74af7-7fbb-48d9-9429-04ebfa0f0b87)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.9

### What app or flow are you having the issue with?

BYODL

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Please go make sure you have the correct Datalake URL parameter from step 9 of [Import Power BI dataflows](https://learn.microsoft.com/power-platform/guidance/coe/setup-powerbi#import-power-bi-dataflows)
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/11a4a36b-71a7-485a-919d-42f40b076cd0)

**Comment by jayakumarkgds** [comment]
Yes i have the exact setup and in preview i can see the apps list.Also dataflow sync status i have posted in my initial description which you can see the number of rows updated for apps.But these are not getting populated in BYODL powerBI dashboard.It was working earlier but in last week sync it removed everything.

**Comment by jayakumarkgds** [comment]
This is fixed by recreating the data flow again.
